antora.yml

```diff
@@ -4,3 +4,6 @@ version: '1.2'
 display_version: 1.2 (Mule 3.9)
 nav:
 - modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    supportStatus: extendedSupport
```

Comment:
Needs to be edited. Want extended support text without telling people to go to the latest version.